src/index.d.ts

```diff
@@ -644,12 +686,69 @@ declare namespace admin.auth {
      */
     tenantId?: string | null;
 
+    /**
+     * The multi-factor related properties for the current user, if available.
+     */
+    multiFactor?: {
+
+      /**
+       * List of second factors enrolled with the current user.
+       * Currently only phone second factors are supported.
+       */
+      enrolledFactors: PhoneMultiFactorInfo[];
```

Comment:
Now that you've changed the other instances in this file, it makes sense to change this to `MultiFactorInfo` too. I think that's safer for users. Otherwise code like the following snippet will break in a future release:

```
const user = await admin.auth().getUser(uid);
if (user.multiFactor) {
  const factor = user.multiFactor.enrolledFactors[0];
  console.log(factor.phoneNumber); // this line will break when we change the type in a future release
}
```

Reply:
Done.